### PR TITLE
Create new sample [issue #24 ]

### PR DESCRIPTION
**Pull request by mi-nakano**
I created new sample for Mixer2 with FakeSMTP. https://nilhcem.github.io/FakeSMTP/
This use Mixer2 to create HTML mail.
The usage is written in README.md.
https://github.com/NakanoMizuki/mixer2-sample/blob/LocalMail-sample/mixer2-localmail/README.md

Since this is provisional snapshot, please comment what to be modified.

### CODE REVIEW

File: mixer2-localmail/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+HTML mail sample with FakeSMTP. see
+https://nilhcem.github.io/FakeSMTP/
+
+Start FakeSMTP.
+```
+java -jar fakeSMTP.x.x.jar
+```
+Set port:8080, directory:any directory, and start server.
```
Comment (nabedge) [inline mixer2-localmail/README.md]: This port number is not compatible to "well-known port".
80: http server
8080: web-ap server
25: mail server

So, this port number shuld be 25 or 1025 (as test purpose mail server)

File: mixer2-localmail/src/main/java/org/mixer2/sample/Main.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.commons.mail.*;
+
+public class Main {
+	private static Logger logger = LoggerFactory.getLogger(Main.class);
+
+	// Server setting
+	private static final String HOST_NAME = "localhost";
+	private static final int PORT = 8080;
```
Comment (nabedge) [inline mixer2-localmail/src/main/java/org/mixer2/sample/Main.java]: port number...

File: mixer2-localmail/src/main/java/org/mixer2/sample/Main.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+	//
+	private static final String TO_ADDR = "nabedge@mixer2.org",
```
Comment (nabedge) [inline mixer2-localmail/src/main/java/org/mixer2/sample/Main.java]: "example" adddres is example.com . i.e, foobar@example.com . see http://www.iana.org/domains/reserved

File: mixer2-localmail/src/main/resources/m2mockup/m2template/mail.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xml:lang="ja" lang="ja" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<title>mail sample for mixer2 and Commons-email</title>
+	<link rel="stylesheet" type="text/css" href="../m2static/css/main.css" media="all" />
```
Comment (nabedge) [inline mixer2-localmail/src/main/resources/m2mockup/m2template/mail.html]: Where is main.css ? (or remove this line)

**Comment by NakanoMizuki** [comment]
Thanks to comment!

* I replaced "PORT=8080" with "PORT_NUMBER=1025"
* I removed "<link ... main.css ... > 

In HTML mail, <head> elements are not sent and only inline style css is usable in <body>.
So I can't use link to main.css.

**Comment by NakanoMizuki** [comment]
Sorry, I forgot to remove css file.

**Comment by NakanoMizuki** [comment]
I modified README.md ( port number)

And I modified Main.java (mail address)

**Pull request merged**